packages/admin/admin/package.json

```diff
@@ -95,6 +96,7 @@
     },
     "peerDependencies": {
         "@apollo/client": "^3.7.0",
+        "@comet/admin-theme": "workspace:^6.4.0",
```

Comment:
This should probably be 6.5.0. Or maybe we could use something like `workspace:*`?